docs/protocol/network/constants.md

```diff
@@ -0,0 +1,36 @@
+# Constants
+
+Constant values are immutable within Witnet protocol versions. A change in the constants necessarily requires a new protocol version.
+
+## Witnet versions
+
+Below the existing versions of the Witnet P2P network protocol are listed:
+
+| Version | Initial Release  | Major Changes |
+| ------- | ---------------- | ------------- |
+| `010`   | to be determined |               |
+
+## Magic numbers
+
+These constant values indicate the originating network in the message headers:
+
+ | Magic number | Network |
+ | ------------ | :-----: |
+ | `0x00`       | mainnet |
+ | `0xF0`       | testnet |
+
+## Node capabilities
+
+Currently, only one node capabilities has been specified in the Witnet network protocol:
+
+| Flag                 | Description                                                        |
+| -------------------- | ------------------------------------------------------------------ |
+| `0x0000000000000001` | NODE_NETWORK, Witnet full node which is the default operation mode |
```

Comment:
It is a 64 bits sequence of masked flag, so that we may support up to 64 services (inspired by Bitcoin specification). Perhaps using hex value is misleading... do you prefer to use a decimal value?

Reply:
I get it. I'm kind of ok with the hex representation, although I think it's non-semantic for big values:
`0x0000000000008000` vs `0000000000000000000000000000000000000000000000001000000000000000`. In the other hand, the binary form is more explicit, yet ludicrously long.

Anyway, I think it needs some explanation about the flags being masks and not explicit values. The point of masks is that they are not exclusive: you could combine `NODE_NETWORK` (`0x0000000000000001`) and an hypothetical `NODE_WHATEVER` (`0x0000000000000002`) by issuing `0x0000000000000003`.